card/mod/05_standard/set/all/rich_html/modal.rb

```diff
@@ -37,7 +37,7 @@ def default_modal_content_args args
   end
 
   view :modal_body do |args|
-    _render_content(args)
+    _render_core(args)
```

Comment:
That actually makes some sense, it's a ruby thing to some extent. run-time vs. load time.  Not the only place this difference might matter.